end-to-end-tests/get-blur-areas.sh

```diff
@@ -36,11 +57,14 @@ check_image() {
 }
 
 # Parameter $1 is the host name where to make the requests to, received by the script.
-check_image $1 "MultiplePeople.jpg" "MultiplePeople.txt"
-check_image $1 "OnePerson.png" "OnePerson.txt"
-check_image $1 "GithubBranching.png" "GithubBranching.txt"
-check_image $1 "RandomText.txt" "RandomText.txt"
-check_image $1 "EmptyFile.txt" "EmptyFile.txt"
+check_image $1 "MultiplePeople.jpg" "MultiplePeople.txt" true false false
+check_image $1 "OnePerson.png" "OnePerson.txt" true false false
+check_image $1 "GithubBranching.png" "GithubBranching.txt" true true true
+check_image $1 "RandomText.txt" "RandomText.txt" true true true
+check_image $1 "EmptyFile.txt" "EmptyFile.txt" true true true
+check_image $1 "OneCarPlate.jpg" "OneCarPlate.txt" false true false
+check_image $1 "GoogleLogo.jpg" "GoogleLogo.txt" false false true
+check_image $1 "CarPlateAndPerson.jpg" "CarPlateAndPerson.txt" true true false
```

Comment:
You could probably add test which returns two rectangles - one for car plates, one for logo, for example - to check that the requests work fine together. I guess in this last test you won't know if everything went fine, or there was some error in the empty request, or something went wrong while adding both requests in the json which you will send.This is why one more test could be useful.

Reply:
The last test contains two rectangles: one for the plate and one for the person, to make sure they work fine together. I can't quite understand what you mean, could you explain it a little bit more? Thank you!

Reply:
I meant two rectangles in the output file. I guess your last test has two requests (for faces and car plates) and one of the responses for those requests is empty. It could probably be useful to also ensure that if it is not empty, nothing still goes wrong.